icefoganalytics/travel-authorization
# Top Selector Make More Visual

Fixes https://github.com/icefoganalytics/travel-authorization/issues/49

# Context

Original info
> The users eye is drawn to the yellow of the details screen.  Perhaps make the outline of the top level selection a different more vibrant colour or a different fill.

## Takeaway After Discussion

- [x] Move the "in territory" selector into the "Purpose" section of the "Details page
- [x] Make the upper content read-only, it should also appear when you are viewing the "Estimates" tab (or any of the other tabs) - this is commonly know as a "layout"

# Screenshots

Edit mode
![image](https://github.com/icefoganalytics/travel-authorization/assets/23045206/50c94fbb-0f86-47d5-baa0-8e9e6cfc067a)

Old edit mode
![image](https://github.com/icefoganalytics/travel-authorization/assets/23045206/5a1025cc-0512-45fa-956a-9cd7d7c568eb)

Read-Only mode
![image](https://github.com/icefoganalytics/travel-authorization/assets/23045206/e72dc4cb-7d98-4c17-b519-c0f9172dc862)

Old read-only mode
![image](https://github.com/icefoganalytics/travel-authorization/assets/23045206/ec6a6ac0-d42b-424c-86d1-f84a417ef6d8)

# Testing Instructions

1. Boot the app via `dev up`
2. Log in to the app at http://localhost:8080/
3. Go to the My Travel Requests page via the top nav dropdown.
4. Create a new travel travel request.
5. Note that the top header panel is now read-only.
6. Note that the "In Territory?" selector is now inside the "Purpose" panel, in front of the "Final Destination" selector.
7. Check that the "In Territory?" selector limits the options in the "Final Destination" selector.
8. Fill in the form and click "Save Draft".
9. Check that the selector value persists.
10. Check that the summary header information also shows up on the "Estimate" tab.
11. Submit the travel authorization to your supervisor, and check that the read-only version of the form has the same style of summary header, and that the "In Territory?" selector is also inside the "Purpose" panel, in front of the "Final Destination" selector.
